omod/src/main/java/org/openmrs/module/messages/resources/ActorResponseRestResource.java

```diff
@@ -132,37 +142,37 @@ public Model getGETModel(Representation rep) {
         ModelImpl model = (ModelImpl) super.getGETModel(rep);
         if (rep instanceof DefaultRepresentation || rep instanceof FullRepresentation) {
             model
-                    .property("id", new StringProperty())
+                    .property("uuid", new StringProperty())
                     .property("patient", new StringProperty())
                     .property("person", new StringProperty())
+                    .property("actorResponseType", new StringProperty())
                     .property("actor", new StringProperty())
                     .property("sourceId", new StringProperty())
                     .property("sourceType", new StringProperty())
                     .property("textResponse", new StringProperty())
                     .property("answeredTime", new DateProperty())
-                    .property("question", new RefProperty("#/definitions/QuestionGetRef"))
-                    .property("response", new RefProperty("#/definitions/ResponseGetRef"))
-                    .property("attributes",
-                            new ArrayProperty(new RefProperty("#/definitions/ActorResponseAttributeGetRef")));
+                    .property("question", new StringProperty())
+                    .property("response", new StringProperty())
+                    .property("attributes",new ArrayProperty(new RefProperty("#/definitions/ActorResponseAttributeGetRef")));
 
         }
         if (rep instanceof DefaultRepresentation) {
             model
-                    .property("preferredResponse", new RefProperty("#/definitions/ResponseGetRef"))
-                    .property("preferredTextQuestion", new RefProperty("#/definitions/TextQuestionGetRef"))
-                    .property("preferredActor", new RefProperty("#/definitions/ActorGetRef"))
-                    .property("preferreQuestion", new RefProperty("#/definitions/QuestionGetRef"));
+                    .property("Response", new RefProperty("#/definitions/ResponseGetRef"))
```

Comment:
Why you changed these properties names and first character to upper case?
Should not these names be a class fields names or can by any names?